src/components/molecules/OcBreadcrumb/OcBreadcrumb.spec.js

```diff
@@ -20,15 +20,49 @@ describe("OcBreadcrumb", () => {
     expect(wrapper.props().variation).toMatch("lead")
     expect(wrapper.classes()).toContain("oc-breadcrumb-lead")
     expect(wrapper).toMatchSnapshot()
-  }),
-    it("displays all items", () => {
-      const wrapper = shallowMount(Breadcrumb, {
-        propsData: {
-          items,
-        },
-      })
+  })
+  it("displays all items", () => {
+    const wrapper = shallowMount(Breadcrumb, {
+      propsData: {
+        items,
+      },
+    })
+
+    expect(wrapper.findAll(".oc-breadcrumb-list-item").length).toBe(4)
+    expect(wrapper.findAll("#oc-breadcrumb-contextmenu-trigger").length).toBe(0)
+    expect(wrapper).toMatchSnapshot()
+  })
+  it("displays context menu trigger", () => {
```

Comment:
Could you group all the context menu tests into a `describe('context menu', () => {` please?

Reply:
```suggestion
  it("displays context menu trigger when a slot is given", () => {
```